app/main/index.ts

```diff
@@ -362,6 +362,21 @@ app.on('ready', () => {
 	ipcMain.on('save-last-tab', (_event: Electron.IpcMessageEvent, index: number) => {
 		ConfigUtil.setConfigItem('lastActiveTab', index);
 	});
+
+	// Update user idle status for each realm after every 15s
+	// Set user idle if no activity for idleThreshold time in seconds
+	// Do a check every idleCheckInterval ms.
+	const idleCheckInterval = 15000;
+	setInterval(() => {
+		const idleThreshold = 60;
+		(electron.powerMonitor as any).querySystemIdleState(idleThreshold, (idleState: string) => {
```

Comment:
This is wrong; why do you need to type cast this to `any`?

Reply:
Apparently, querySystemIdleState has not been added yet to types of powerMonitor. tsc gives the following error-
```
app/main/index.ts:372:25 - error TS2339: Property 'querySystemIdleState' does not exist on type 'PowerMonitor'.

372  	electron.powerMonitor.querySystemIdleState(idleThreshold, (idleState: string) => {
     	                      ~~~~~~~~~~~~~~~~~~~~


Found 1 error.

```

Reply:
OK, Add a comment here so we can come back to it in future.

Reply:
Done.